spek-runtime/jvm/src/main/kotlin/org/spekframework/spek2/runtime/SpekJvmRuntime.kt

```diff
@@ -38,15 +38,13 @@ actual class SpekRuntime: AbstractRuntime() {
             .filter { !it.isAbstract }
             .map { klass ->
                 klass to PathBuilder.from(klass).build()
+            }.map { (klass, path) ->
```

Comment:
Let’s follow the same style everywhere here, i. e.

```kotlin
.map { 
}
.filter { }
.map { }
```

Reply:
👍